lib/v2/bilibili/cache.js

```diff
@@ -1,30 +1,62 @@
 const got = require('@/utils/got');
+const config = require('@/config').value;
 
 module.exports = {
+    getCookie: (ctx) => {
+        const key = 'bili-cookie';
+        return ctx.cache.tryGet(key, async () => {
+            const response = await got(`https://www.bilibili.com/`, {
+                headers: {
+                    'User-Agent': config.UA
```

Comment:
No need to set UA again as it's already set in `lib/utils/got`
https://github.com/DIYgod/RSSHub/blob/3e6f0542c2ae51e9650376f4c435c109e548a85d/lib/utils/got.js#L33-L35